lib/skate_web/templates/admin/index.html.eex

```diff
@@ -0,0 +1,8 @@
+<h2>Admin Tools</h2>
```

Comment:
nitpick(non-blocking): Are there other headers on the page "preventing" this from being `<h1>`?

Reply:
Thanks for noticing!